docs/api-provider-service.yaml

```diff
@@ -0,0 +1,506 @@
+---
+openapi: 3.0.0
+
+info:
+  version: '1.0.0'
+  title: 'Provider Service API'
+  description: >-
+          Defines resources and methods that tenants of OpenStack can
+          use for trading servers with the Marketplace.
+
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+
+
+# Added by API Auto Mocking Plugin
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/FLOCX/Provider_Service/1.0.0
+
+tags:
+  - name: offer
+    description: Operations about selling servers
+    externalDocs:
+      description: To render copy this file into
+    url: https://editor.swagger.io/
+  - name: contract
+    description: Operations about Contracts
+    externalDocs:
+      description: To render copy this file into
+    url: https://editor.swagger.io/
```

Comment:
indent

Reply:
I fixed this.
Surprisingly, when I test it on my local branch using yamllint, it does not catch these errors.